addons/point_of_sale/static/src/app/store/cash_opening_popup/cash_opening_popup.xml

```diff
@@ -8,7 +8,7 @@
                     <Input tModel="[state, 'openingCash']"
                         isValid.bind="env.utils.isValidFloat"
                         callback.bind="handleInputChange"
-                        autofocus="true" />
+                        autofocus="true" placeholder="'Opening Balance Eg: 123'"/>
```

Comment:
> 

Hello @paso-odoo , the way meji had the string wrapped in quotes is important in this case. This is because Owl will evaluate the content inside the strings as a js expression. 
For example, if i have `placeholder="myPlaceholder"`, owl will try to find the value of the js expression `myPlaceholder`; if, on the other hand, i have `placeholder="'myPlaceholder'"`, owl will simply render the string `myPlaceholder`.



In any case, the concern about formatting is valid.

Reply:
@vlst-odoo 

Got that
Thanks for the info!